jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -213,15 +231,91 @@ data class CodeModernizerSessionContext(
                     reason = error
                 )
                 LOG.warn { error }
-                return null
```

Comment:
when there is no maven install in customers local machine, we can still use the IDE bundled maven to build. So delete this line so we can go forward to IDE bundled maven.